ckanext/who_afro/blueprints/dashboard_extension.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+from flask import Blueprint, render_template
+from ckanext.activity.views import dashboard
+
+log = logging.getLogger(__name__)
+
+blueprint = Blueprint(
+    "dashboard_extension", __name__, url_prefix="/dashboard"
+)
+
+
+# Returns dashboard but with a different request.path
+@blueprint.get("/user_activity", endpoint="user_activity")
+def dashboard_index():
+    return dashboard()
```

Comment:
If the entire purpose of this blue print is just to change the url of one page, I don't think it is worth it?  Can we just stick with the original URL structure?

Reply:
Updated.

Reply:
I have removed the blueprint and instead I have added additional arguments to the URLs of each of the tab items: 
```
        {{ h.build_nav_icon('activity.dashboard', _('Newsfeed')) }}
        {{ h.build_nav_icon('dashboard.datasets', h.humanize_entity_type('package', dataset_type, 'my label') or _('My datasets')) }}
        {{ h.build_nav_icon('dashboard.organizations', h.humanize_entity_type('organization', org_type, 'my label') or _('My teams')) }}
        {{ h.build_nav_icon('activity.dashboard', _('My activity'), type='user', name=user.name, id="user_activity") }}
```

Then I capture the argument (if it exists and with value) then it is the 'my activity page':

In user/dashboard.html: 

```
{% block primary_content_inner %}
    {% if request.args.get("id", "") == 'user_activity' %}
        {% snippet 'snippets/stream.html', activity_stream=dashboard_activity_stream, id=id, object_type='user' %}
        {% snippet 'snippets/pagination.html', newer_activities_url=newer_activities_url, older_activities_url=older_activities_url %}
    {% else %}
        {{ super() }}
    {% endif %}
{% endblock %}
```